crowdin.yml

```
@@ -18,6 +18,7 @@
         # map non-two letter codes here, left side is crowdin designation, right side is babel designation
         "en-GB": "en_GB",
         "en-US": "en_US"
+        "zh-TW": "zh_TW"
```

```suggestion
        "en-US": "en_US",
        "pt-BR": "pt_BR",
        "zh-TW": "zh_TW"
```

another language was missing here as well. Also need to add a comma to the previous entries.

---

Part of this suggestion was missed. Can you please add pt-BR?

---

Done.